migrator/migrations/m_98_to_99_violation_report_branding/migration.go

```diff
@@ -0,0 +1,50 @@
+package m98to99
+
+import (
+	"embed"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/migrator/migrations"
+	"github.com/stackrox/rox/migrator/migrations/policymigrationhelper"
+	"github.com/stackrox/rox/migrator/types"
+	bolt "go.etcd.io/bbolt"
+)
+
+var (
+	migration = types.Migration{
+		StartingSeqNum: 98,
+		VersionAfter:   storage.Version{SeqNum: 99},
+		Run: func(databases *types.Databases) error {
+			err := updatePolicies(databases.BoltDB)
+			if err != nil {
+				return errors.Wrap(err, "updating policies")
+			}
+			return nil
+		},
+	}
+
+	//go:embed policies_before_and_after
+	policyDiffFS embed.FS
+
+	// We will want to migrate only if the existing policy sections and title haven't changed.
+	fieldsToCompare = []policymigrationhelper.FieldComparator{
+		policymigrationhelper.PolicySectionComparator,
+		policymigrationhelper.NameComparator,
```

Comment:
Also add `policymigrationhelper.DescriptionComparator`. This migration is changing the description as well and we don't want to change it if the user has changed it.